velox/vector/tests/VectorTest.cpp

```diff
@@ -3101,5 +3101,240 @@ TEST_F(VectorTest, appendNulls) {
   // Append negative.
   EXPECT_ANY_THROW(rowVector->appendNulls(-1));
 }
+
+TEST_F(VectorTest, debug) {
```

Comment:
debug ?

Reply:
Sorry that I forgot to remove it, will remove it.